testdata/errors.go

```diff
@@ -31,6 +31,12 @@ func g(x int) error {
 		return fmt.Errorf("This %d is too low", x) // MATCH /error strings.*not be capitalized/
 	} else if x == 0 {
 		return fmt.Errorf("XML time") // ok
+	} else if x > 3 {
+		return fmt.Errorf("newlines are fun\n") // MATCH /error strings.*not end with a newline/
+	} else if x > 2 {
+		return fmt.Errorf("Newlines are really fun\n") // MATCH /error strings.*or end with a newline/
+	} else if x > 1 {
+		return fmt.Errorf("newlines are\nespeciallyfun\non a console\n")
 	}
 	return errors.New(`too much stuff.`) // MATCH /error strings.*not end with punctuation/
```

Comment:
I second that, it should be more readable even if it is just a test file that is never executed. I was quite confused by looking at the whole file. The non-checking comments do not make much sense either.

@shurcooL btw the previous version had a case for `x < 1` then `x == 0` (dead code). The dangling else is therefore `x >= 1`